kmapper/visuals.py

```diff
@@ -58,6 +58,28 @@
 ]
 
 
+def colorscale_from_matplotlib_cmap(cmap, ii_off=0, ff_off=0, nbins=10):
+    """Create a colorscale from a matplotlib colormap.
+    see https://matplotlib.org/tutorials/colors/colormaps.html
+
+    Example:
+      import matplotlib.pyplot as plt
+      colorscale = colorscale_from_matplotlib_cmap(plt.cm.cool)
```

Comment:
Could you add your explanation of what `ii_off` and `ff_off` are within a comment here. You can format the params like the others: https://github.com/scikit-tda/kepler-mapper/blob/51dde7d0469feeb7d76ba1932bae4ba7cc71e970/kmapper/kmapper.py#L87